model-engine/model_engine_server/infra/gateways/fake_monitoring_metrics_gateway.py

```diff
@@ -57,3 +62,6 @@ def emit_database_cache_hit_metric(self):
 
     def emit_database_cache_miss_metric(self):
         self.database_cache_miss += 1
+
+    def emit_route_call_metric(self, route: str, _metadata: MetricMetadata):
```

Comment:
Maybe outside of the scope of this PR, but might be good to adopt Datadog-esque terminology, where `increment` means += 1. `emit` to me sounds more like a gauge or count.

Reply:
Do you have a link to the "Datadog-esque terminology" you are referring to? What is currently here looks like it is just pattern matching the `emit_*` pattern.

Reply:
Yeah was thinking of https://docs.datadoghq.com/metrics/custom_metrics/dogstatsd_metrics_submission/
![image](https://github.com/scaleapi/llm-engine/assets/2041983/96af51b2-4a53-4d1b-9f95-b88091b286e8)

But yeah acknowledge that this is just following the existing naming pattern (which is what I think could be tweaked).